Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM jfloff/alpine-python:3.6
```

Comment:
let's use the official python image here, unless there's a specific reason that you're using the jfloff one

Reply:
It's a third of the size.

Reply:
Ah, I just checked out the official python image and it does have alpine images. I'll test those